src/app/shared/input/input.component.ts

```diff
@@ -10,4 +10,10 @@ export class InputComponent {
   @Input() name = '';
   @Input() value = '';
   @Output() valueChange = new EventEmitter<string>();
+
+  handleTyping(event: Event): void {
+    const target = event.target as HTMLInputElement;
+    const value = target.value;
+    this.valueChange.emit(value);
+  }
```

Comment:
why not just 
```
handleTyping(event: Event): void {
    this.valueChange.emit(event.target.value)
  }
```
?

Reply:
My mistake

Reply:
When I use 
handleTyping(event: Event): void {
    this.valueChange.emit(event.target.value)
  }
  Ide show me objet is possibly null and error target Property 'value' does not exist on type 'EventTarget'.